uploadcare.js

```diff
@@ -52,7 +57,9 @@
                                 imageUrl += '-/preview/';
                             }
                             if (this.isImage) {
-                                $this.insert.html('<img src="' + imageUrl + '" alt="' + this.name + '" />', false);
+                                var openTag = $opts.imageTag ? '<' + $opts.imageTag + '>' : '';
```

Comment:
Yes, it's `figure` according to the [docs](https://imperavi.com/redactor/docs/settings/images/#setting-imageTag), so it should never be empty.